### ISSUE TITLE
Does every new animation need to be retrained?

### ISSUE THREAD

**Issue by yize0106** [showlab/MotionDirector#35]
Hello, Thanks for releasing the nice work.
I have just come into contact with this field, and I have a question that I am curious about: If I want to animate new image with different appearances and customize new actions, do I need to use the image and related videos to train again?

If so, does it mean that the weights given by [link](https://huggingface.co/ruizhaocv/MotionDirector/tree/main/image_animation) were trained only on the reference image/video and no other new images/videos were used?

Looking forward to your answer very much！

**Comment by ruizhaocv** [comment]
Yes, for new motions, you need to train the LoRAs again. This is a typical setting in the tuning-based customization task.
